src/components/VarianceTypeActionsPopover.vue

```diff
@@ -23,19 +17,62 @@ import {
   IonItem,
   IonList,
   IonListHeader,
+  popoverController
 } from "@ionic/vue";
 import { defineComponent } from "vue";
 import { translate } from '@hotwax/dxp-components'
+import { mapGetters, useStore } from "vuex";
+import { UtilService } from "@/services/UtilService";
+import logger from "@/logger";
+import { hasError } from "@/adapter";
+import { showToast } from "@/utils";
+
 export default defineComponent({
-  name: "RejectReasonActionsPopover",
+  name: "VarianceTypeActionsPopover",
   components: {
     IonContent,
     IonItem,
     IonList,
     IonListHeader
   },
+  computed: {
+    ...mapGetters({
+      rejectReasons: 'util/getRejectReasons',
+      rejectReasonEnumTypes: 'util/getRejectReasonEnumTypes'
+    })
+  },
+  props: ["reason"],
+  methods: {
+    async updateVarianceType(selectedType: any) {
+      try {
```

Comment:
Handle the case when the same variance type is clicked that is already selected.